src/main/java/org/sagebionetworks/web/client/widget/entity/browse/EntityFinderWidgetImpl.java

```diff
@@ -63,15 +64,15 @@ public EntityFinderWidgetImpl(final EntityFinderWidgetView view, final GlobalApp
         view.setPresenter(this);
     }
 
-    private EntityFinderWidgetImpl(Builder builder) {
+    private EntityFinderWidgetImpl(final Builder builder, final PortalGinInjector ginInjector) {
         this.selectedEntities = new ArrayList<>();
 
-        // Pass along dependencies injected into the builder because they are not injected in this constructor.
-        this.view = builder.view;
```

Comment:
Bug was caused by the builder re-using the same view instance for each built EntityFinderWidget

Reply:
This is a nice simplification of the code, let the GIN (Guice) binding determine which objects should be singletons